src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -311,11 +310,11 @@ Before you enable sign-up, sign-in, and sign-out there are a few decisions you w
 #### Which Amazon Cognito attributes will you pass for sign-up?
 
 User attributes (e.g. username, email, etc.) are used to identify your users. These are set up when you configure your Cognito User Pool and cannot be renamed or deleted after they are configured. You will need to know which attributes were configured as required at set up (e.g. email) and make sure your user is prompted to enter this information. You can find this information in the Amazon Cognito console under the *Sign-up experience* tab. You can also add [additional User Pool attributes]( https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html) that are collected from the sign-up form. This includes attributes like address, birthdate, phone, etc. These attributes are not required by default. You can make these additional attributes required but it cannot be changed back to not required once configured.
-
+ 
 #### How will you verify users at sign-up?
 
 Once the initial sign-up is done, the user will need to verify and confirm their contact information before their account is activated. This is done by sending a verification code via email or phone number that can be collected on initial sign-up. Email verification asks the user to click a link or enter a code sent to their email address provided during sign-up. Phone verification is done by asking the user to enter the code sent to them in a SMS text message. You can also enable auto-confirmation to skip this step and automatically confirm the user after sign-up. This is accomplished using the PostSignUp Lambda trigger.
-
+ 
```

Comment:
Is there a specific link we should use for the PostSignUp Lambda trigger - also should that PostSignUp be listed as code or in quotes?

Reply:
it should be a code snippet. I think we should use the `postConfirmation` Lambda though. What do you think @nadetastic https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-post-confirmation.html

Reply:
It should be `PreSignUp` lambda that allow you to auto-confirm a user, and the result is it doesn't send a confirmation email to the user

- https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-pre-sign-up.html

Reply:
Updated!